gradle/build.versions.toml

```diff
@@ -0,0 +1,42 @@
+[versions]
+asm = "9.3"
+jackson = "2.13.2"
+junit5 = "5.7.1"
+spock = "2.1-groovy-3.0"
+
+[libraries]
+ant = "org.apache.ant:ant:1.10.8"
```

Comment:
Is there a reason we can just use what's provided by `gradleApi()`?

Reply:
there was a time where we needed a newer version of ant than that was shipped with gradle.

Reply:
Ah yes. our UnzipTransforms rely on `org.apache.tools.zip` that is not part the gradle api